server/public/styles.css

```diff
@@ -75,10 +75,94 @@ body {
   text-align: center;
   background-color: #ffff64;
   overflow: hidden;
+  display: flex;
+  justify-content: center;
+}
+
+.yellow-contents {
+  width: calc(100% / 3 * 2);
+  margin-bottom: 2.5rem;
+}
+
+.session-data {
+  width: 100vw;
+  display: flex;
+  flex-direction: column;
+}
+
+.session-data > div {
+  margin-left: 1rem;
+  margin-right: 1rem;
+}
+
+.session-stats > div {
+  display: flex;
+  justify-content: center;
+}
+
+.session-stats > div > p {
```

Comment:
Bootstrap has lots of margin helper classes. You should favor those. Plus, child selector hierarchies like this are very brittle.